### PR TITLE
cNFT Payout Royalties

### PR DESCRIPTION
**Pull request by devey**
Moved `Payout` and `Royalty` logic from `NFT`.

### CODE REVIEW

**Review by achrafismyname — APPROVED** [review]
Seems good 👍

File: as/cNFT/assembly/royalty_payout.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { u128 } from 'near-sdk-as'
+import { royalty_to_payout } from '../../utils'
```
Comment (AshenI99) [inline as/cNFT/assembly/royalty_payout.ts:2]: Here you have imported `royalty_to_payout` method from the `as/utils.js` file in root folder. But you have already added a `royalty_to_payout` method  in `as/cNFT/assembly/utils/royalties.ts` in too. So I think you should import it from that one.

File: as/cNFT/assembly/royalty_payout.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import { persistent_tokens } from './models/persistent_tokens'
+import { persistent_tokens_royalty } from './models/persistent_tokens_royalty'
+import { Payout } from './types'
+
+@nearBindgen
+export function nft_payout(
+    token_id: string,
```
Comment (AshenI99) [inline as/cNFT/assembly/royalty_payout.ts:9]: `token_id` shoud be in type `TokenId`.

**Review by AshenI99 — APPROVED** [review]
Other than two minor issues, this one seems good to me. :-)

File: as/cNFT/assembly/royalty_payout.ts (hunk 1 above)
Comment (devey) [reply as/cNFT/assembly/royalty_payout.ts:2]: Correct!

**Pull request merged**